gateway/src/api/latest.rs

```diff
@@ -267,12 +267,26 @@ async fn delete_project(
 
     let project_name = scoped_user.scope.clone();
     let project = state.service.find_project(&project_name).await?;
+
     let project_id =
         Ulid::from_string(&project.project_id).expect("stored project id to be a valid ULID");
 
-    // Try to startup destroyed or errored projects
+    // Try to startup destroyed, errored or outdated projects
     let project_deletable = project.state.is_ready() || project.state.is_stopped();
-    if !(project_deletable) {
+    let lowest_restartable_version = semver::Version::new(0, 39, 0);
```

Comment:
We should just compare it to the current version of gateway IMO

Reply:
So restart everything less than 0.41 you mean?

Reply:
Yeah everything less than `env!("CARGO_PKG_VERSION")`

Reply:
Yeh, good point. Given projects will be deleted, it doesn't matter if we restart it and it turns out the deployment will crash for any reason (given the app is supposed to be stopped and deleted with all its resources).

Reply:
Addressed here: https://github.com/shuttle-hq/shuttle/pull/1677/commits/f767676db566e550bef54d7e09b0d3f7528c1df0. What do you think?